### DISCUSSION TITLE
SUMMARY for Daily Notes template (auto fill FROM and TO date range for template use)

### DISCUSSION THREAD

**Discussion by trent-swanson** [mcndt/obsidian-toggl-integration#157] [Feature Requests]
I have been enjoying this plugin as part of my workflow, however I noticed an issue with my use case.

I have a toggl summary as part of my template for Daily Notes.
However, the summary will show the current day or the Past 7 Days from today, and not from the date of the note.
Such that if I review a daily note from earlier in the year, it will still show only the current week, and not the week of the daily note.

I would love for the summary to display the Day and Week of the date of the daily note (either from date of creation or the title of the note: YYYY-MM-DD), without having to manually enter in FROM - TO dates.

Thanks.

**Comment by ajilityj** [comment]
@trent-swanson you may find [this discussion](https://github.com/mcndt/obsidian-toggl-integration/discussions/123) helpful

**Comment by trent-swanson** [comment]
Using **Periodic Notes** I was able to get weekly reports for the week of the note. 
And also monthly reports for the month of the note. 

FROM {{sunday:YYYY-MM-DD}} TO {{saturday:YYYY-MM-DD}}
FROM {{date:YYYY-MM-DD}} TO {{date+31d:YYYY-MM-DD}}

Added to note template and auto filled upon note creation.
